### ISSUE TITLE
Make Contributing section of README more proheminent

### ISSUE THREAD

**Issue by LTKills** [LTKills/heroes_build_scrapper#44]
One of the biggest problems I've had with contributions is that they did not follow the instructions on the Contributing section of README. This is problematic becaus everytime someone opens a PR to an issue they were not assigned to and (this is the actual problem) it does not tackle the problem correctly most of the time because the person who wrote the PR did not discuss details of implementation or code standards in the issue before rushing the code.

I'm looking for a way to make the contributions' guidelines more visible and important, so as to avoid problematic PRs such as the ones cited above.

**Comment by ew4321** [comment]
What if I added a pull request template that:

1. Asks contributors to add themselves to the PLAYERS.md file
2. Asks contributors to check if the issue is currently open and assigned to them
3. Asks contributors to check that their PRs and commits contain meaningful/descriptive titles 

Anything else?

**Comment by LTKills** [comment]
No, but that has already been done. What do you propose to make it more proheminent? Maybe making a file for these instructions (`Contributing.md`) and adding a reference to it in the README, but I'm not sure this would make it *more* proheminent.
